### ISSUE TITLE
I’m the maintainer of codice_fiscale

### ISSUE THREAD

**Issue by eutampieri** [succoDiPompelmo/codice_fiscale_rs#7]
…and I’m open to discuss moving your crate to the codice_fiscale name.
Or anything else you wish :). I think though that two crates for CF validation/generation are too much.

**Comment by succoDiPompelmo** [comment]
Hello @eutampieri :wave:. I agree on you point it's a bit of a waste to have two crates. Do you have any suggestion on how to merge them ? Do you think we should simply rename this one ?

One point of friction that I see is about Belfiore code, do we want to import them here ?

In my mind I would of course bridge all the tests on this crate to be sure we cover all the cases. Then if we want to Import Belfiore codes a simple PR should be enough. Changing name on `crates.io` didn't know it was possible, but if it is I think we can go for that way.

**Comment by eutampieri** [comment]
Yes, I agree with your Belfiore code considerations. I’ll have to think how to do this. Perhaps I could depend on your crate in the one I maintain and then implement there the missing features.

**Comment by succoDiPompelmo** [comment]
I think the goal should be to have 2 library, one dedicated to `belfiore` computation and storing the updated municipalities list and the other one focused on handling the `codice fiscale`, but decoupled between each other. This at least was my initial goal but sadly it's hard to have updated and correct `belfiore` codes available.
In addition many alternative libraries in other languagues made the same choice to avoid the internal computation and let client provide the code.

If you think it's acceptable we can drop the `belfiore` code on your library and make it a separate crate, my wet dream was to be able to update the codes list with automation, and see if we can marge the remaining parts ?

"Sorry for the slow response but no much free time at work in this period"

**Comment by succoDiPompelmo** [comment]
If interested this was I was planning to have for Belfiore codes in the library https://github.com/lucavandro/CodiceFiscaleJS/blob/master/update_comuni.py

**Comment by eutampieri** [comment]
I think we can definitely collaborate on this, even though I won't have time until at least the second half of September. You can then move your crate to the new name and I can then write the Belfiore crate.

**Comment by succoDiPompelmo** [comment]
Definitly, I could start to do some preliminary work on both crates if I find some free time in the following weekends.
